src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/HttpLoggingTagNames.cs

```diff
@@ -34,12 +34,12 @@ public static class HttpLoggingTagNames
     /// <summary>
     /// HTTP Request Headers prefix.
     /// </summary>
-    public const string RequestHeaderPrefix = "RequestHeader.";
+    public const string RequestHeaderPrefix = "http.request.header.";
 
     /// <summary>
     /// HTTP Response Headers prefix.
     /// </summary>
-    public const string ResponseHeaderPrefix = "ResponseHeader.";
+    public const string ResponseHeaderPrefix = "http.response.header.";
```

Comment:
New name [http.request.header.](https://github.com/open-telemetry/semantic-conventions/blob/b127e121b9b458e83a0fee8a32f7eb6721912951/docs/http/http-spans.md#common-attributes)

Reply:
See the comment above

Reply:
Wait, the link you shared is related to spans. Is it applicable to logs?